ini_python_multiline_test.go

```diff
@@ -0,0 +1,50 @@
+package ini_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+}
+
+func TestMultiline(t *testing.T) {
+	Convey("Parse Python-style multiline values", t, func() {
+
+		// enable Python-style multiline values
+		opts := ini.LoadOptions{
+			AllowPythonMultilineValues: true,
+		}
+
+		// load test data
+		path := filepath.Join("testdata", "multiline.ini")
+		f, err := ini.LoadSources(opts, path)
+
+		// Should have no error
+		So(err, ShouldBeNil)
+
+		// Should have parsed data
+		So(f, ShouldNotBeNil)
+
+		// Should have only the default section
+		So(len(f.Sections()), ShouldEqual, 1)
+
+		// Should have default section
+		defaultSection := f.Section("")
+		So(f.Section(""), ShouldNotBeNil)
+
+		// Default section should map to test data struct
+		var testData testData
+		err = defaultSection.MapTo(&testData)
+		So(err, ShouldBeNil)
+
+		// Parsed values should match expected values
+		So(testData.Value1, ShouldEqual, "some text here\nsome more text here\n\nthere is an empty line above and below\n")
```

Comment:
If you think using `\n` is not very obvious, you could use back quotes:

```go
So(testData.Value1, ShouldEqual, `some text here
some more text here

there is an empty line above and below
`
```

Reply:
I wanted the tests to be in a single line. Now that I added a new test case with very large text, I used this method above for that. I am not sure if you are asking me to change the other tests to follow the same pattern. To make it simple, I will go ahead and change them as well just in case to close out this pull request